routes/admin.go

```diff
@@ -1,207 +1,28 @@
-	ctx := r.Context()
-	event := ctx.Value(ContextEventKey).(*models.Event)
-
-	updatedEvent := UpdatedEvent{}
-	err := json.NewDecoder(r.Body).Decode(&updatedEvent)
-	if err != nil {
-		w.WriteHeader(http.StatusBadRequest)
-		logs.Error(err)
-		return
-	}
-
-	event.Name = updatedEvent.Name
-	event.Author = updatedEvent.Author
-	event.Location = updatedEvent.Location
-	if updatedEvent.Date != nil {
-		date := models.Timestamp(time.Unix(*updatedEvent.Date, 0))
-		event.Date = &date
-	}
-
-	err = orm.GET.Events.Save(event)
-	if err != nil {
-		w.WriteHeader(http.StatusBadRequest)
-		logs.Error("Failed to save an updated event: ", err)
-		return
-	}
-
-	remote.BroadcastState()
-}
-
-func serveImage(w http.ResponseWriter, r *http.Request) {
-	ctx := r.Context()
-	event := ctx.Value(ContextEventKey).(*models.Event)
-	image := ctx.Value(ContextImageKey).(*models.Image)
-
-	subpath := "pictures"
-	if image.Unattended {
-		subpath = "unattended"
-	}
-
-	path := utils.GetPath("image", fmt.Sprintf("%v", event.Id), "images", subpath, fmt.Sprintf("%v", image.Id)) + ".jpg"
-	if _, err := os.Stat(path); os.IsNotExist(err) {
-		WriteError(w, errors.New("image not found"), http.StatusNotFound, "Can't find file on filesystem")
-		return
-	}
-
-	http.ServeFile(w, r, path)
-}
-
-func getLastExports(w http.ResponseWriter, r *http.Request) {
-	values := mux.Vars(r)
-	event_id := values["event_id"]
-
-	eventId, err := strconv.ParseInt(event_id, 10, 64)
-	if err != nil {
-		w.WriteHeader(http.StatusBadRequest)
-		return
-	}
-
-	event, err := orm.GET.Events.GetEvent(eventId)
-	if err != nil {
-		// @Todo: handle better
-		w.WriteHeader(http.StatusNotFound)
-		return
-	}
-
-	exportedEvents, err := orm.GET.Events.GetExportedEvents(event, 5)
-	if err != nil {
-		w.WriteHeader(http.StatusInternalServerError)
-		logs.Error(err)
-		return
-	}
-
-	data, _ := json.MarshalIndent(exportedEvents, "", "  ")
-
-	w.Header().Set("Content-Type", "application/json")
-	w.Write(data)
+	// r.HandleFunc("/set_mode/{mode}", setMode).Methods(http.MethodPost) // Unused, but @TODO since we want to remove the most from WS
 }
 
-func downloadExport(w http.ResponseWriter, r *http.Request) {
-	values := mux.Vars(r)
-	exportedEventStr := values["export_id"]
-
-	exportedEventId, err := strconv.ParseInt(exportedEventStr, 10, 64)
-	if err != nil {
-		w.WriteHeader(http.StatusBadRequest)
-		return
-	}
-
-	exportedEvent, err := orm.GET.Events.GetExportedEvent(exportedEventId)
-	if err != nil {
-		w.WriteHeader(http.StatusNotFound)
-		logs.Error(err)
-
-		return
-	}
-
-	path := utils.GetPath("images", fmt.Sprintf("%v", exportedEvent.EventId), "exports", exportedEvent.Filename)
-	if _, err := os.Stat(path); os.IsNotExist(err) {
-		WriteError(w, errors.New("export not found"), http.StatusNotFound, "Can't find file on filesystem")
-		return
-	}
-
-	http.ServeFile(w, r, path)
+// @TODO: Oops
+func isPasswordValid(c echo.Context) error {
+	return c.String(http.StatusOK, "yes")
```

Comment:
Not implemented as we're moving to user-based system soon, we won't need this anymore